web/app.js

```diff
@@ -1512,6 +1513,36 @@ if (typeof PDFJSDev === 'undefined' || PDFJSDev.test('GENERIC')) {
   };
 }
 
+function loadFakeWorker() {
+  return new Promise(function(resolve, reject) {
+    if (typeof PDFJSDev === 'undefined' || !PDFJSDev.test('PRODUCTION')) {
+      if (typeof SystemJS === 'object') {
+        SystemJS.import('pdfjs/core/worker').then((worker) => {
+          window.pdfjsNonProductionPdfWorker = worker;
+          resolve();
+        });
+      } else if (typeof require === 'function') {
+        window.pdfjsNonProductionPdfWorker = require('../core/worker.js');
+        resolve();
+      } else {
+        reject(new Error(
+          'SystemJS or CommonJS must be used to load fake worker.'));
+      }
+    } else {
+      let script = document.createElement('script');
+      script.src = PDFJS.workerSrc;
+      script.onload = function() {
+        resolve();
+      };
+      script.onerror = function () {
+        reject(new Error(`Cannot load fake worker at: ${script.src}`));
+      };
+      (document.getElementsByTagName('head')[0] || document.body).
+        appendChild(script);
+      }
```

Comment:
Nit: this bracket is misindented.

Reply:
Whoops, too much copy-pasting of existing code here (and above as well); good eye!